doc/tutorial_classification.rst

```diff
@@ -6,4 +6,164 @@
 Tutorial
 ========
 
-TO BE CONTINUED
\ No newline at end of file
+In this tutorial, we compare the prediction sets estimating by Mapieclassifier in a two-dimensional dataset with three labels. The distribution of the data is a bivariate normal with diagonal covariance matrices for each label. 
+
+Throughout this tutorial, we will answer the following questions:
+
+How do the number of class in the prediction sets vary according to the values ​​of alpha with the Conformal Prediction method ?
```

Comment:
Ajouter un - avant la question (il n'y en aura plusieurs)

Reply:
Renommer la question en 
"How do the number of classes in the prediction sets vary according to the significance level alpha values and the conformity scores ?"

Reply:
modifié